libs/formAnswers.js

```diff
@@ -1,4 +1,4 @@
-function generateDataMap(form) {
+export function generateDataMap(form) {
```

Comment:
Suggestion here is to rename this function to something along the lines with getFormFieldsToPopulate() or getFormFieldsWithLoadPreviousAttribute()

Reply:
Will keep for now

Reply:
Is this not a quick fix? That would improve readability? And when will these changes get addressed otherwise?

Reply:
Done!